.github/workflows/publish_v1_binaries.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish v0.x Binaries
```

Comment:
v0.x ?

Reply:
This needs to be v1.x! Thanks for pointing it out ☺️